tools/azsdk-cli/Azure.Sdk.Tools.Cli/Services/Languages/JavaLanguageSpecificChecks.cs

```diff
@@ -111,31 +111,34 @@ public async Task<CLICheckResponse> LintCodeAsync(string packagePath, bool fixCh
                 return prerequisiteCheck;
             }
 
-            // Use Azure SDK approach: mvn install with linting properties (based on run-and-validate-linting.yml)
-            // This matches the Azure SDK for Java pipeline which runs linting during install phase
+            // Use mvn install with ALL linting tools in fail-safe mode
+            // This follows the "accumulate all errors" pattern instead of failing fast
+            // The -am flag ensures parent POMs are built/resolved automatically
             var command = "mvn";
             var args = new List<string>
             {
                 "install",
                 "--no-transfer-progress",
                 "-DskipTests",
                 "-Dgpg.skip",
-                "-Dmaven.javadoc.skip=true",
+                "-DtrimStackTrace=false",
```

Comment:
I'm wondering if this has any overlap with current `build` command.

Reply:
Is the build command located centrally? I was assuming in the `build` command we would be skipping linting, but if not we can definitely reuse.